Makefile

```diff
@@ -73,7 +73,7 @@ bin-docker-machine: mkdir
 bin-kompose: mkdir
 	curl -Lo $(DIR)/bin/kompose \
 		https://github.com/kubernetes/kompose/releases/download/v1.21.0/kompose-$(VENV_OS)-amd64
-	chmod 755 $(DIR)/bin/kubectl
+	chmod 755 $(DIR)/bin/kompose
```

Comment:
already addressed in branch `develop`. This will be merged into stable in another PR

Reply:
@shaikh-rashid : I don't think this is addressed in `develop` yet: 

https://github.com/CanDIG/CanDIGv2/blob/develop/Makefile#L73-L76